fixtures/plain.js

```diff
@@ -0,0 +1 @@
+console.log('Javascript');
```

Comment:
You could also use the fixture `noop` that already exists.

Reply:
Yes, but I wanted to try that fork use the same process as the parent, ie `node`. `noop` is a bash script.

Reply:
`noop` is not a Bash script, it's a Node.js file with a shabang. Maybe we're talking about a different file: https://github.com/sindresorhus/execa/blob/master/fixtures/noop

Fork will never use the same process as the parent, it will always run in a child process.